ilc/server/tailor/merge-configs.spec.js

```diff
@@ -90,6 +90,54 @@ describe('merge configs', () => {
                 },
             },
         },
+        {
```

Comment:
Pls create separate tests for route replacement & beforeOf. Now this single suite does too many checks...

Reply:
Added separated tests for it